ember_debug/libs/render-tree.js

```diff
@@ -67,10 +67,30 @@ class InElementSupportProvider {
 
     const captureNode = this.debugRenderTree.captureNode;
     this.debugRenderTree.captureNode = function (...args) {
-      const capture = captureNode.call(this, ...args);
+      const nodeFor = this.nodeFor;
       const [id, state] = args;
       const node = this.nodeFor(state);
-      self.setupNodeRemotes(node, id, capture);
+      let capture;
+      try {
+        capture = captureNode.call(this, ...args);
+        self.setupNodeRemotes(node, id, capture);
+      } catch (e) {
+        this.nodeFor = () => {
+          this.nodeFor = nodeFor;
```

Comment:
Can you please explain this? I'm confused why we would set this to a function that then sets itself to something else immediately.

Reply:
CaptureNode will call this function also for children nodes, so need to reset to original immediately